edb/server/main.py

```diff
@@ -487,13 +509,18 @@ def _protocol_version(
 
 
 def _validate_max_backend_connections(ctx, param, value):
-    if value < defines.BACKEND_CONNECTIONS_MIN:
+    if value is not None and value < defines.BACKEND_CONNECTIONS_MIN:
         raise click.BadParameter(
             f'the minimum number of backend connections '
             f'is {defines.BACKEND_CONNECTIONS_MIN}')
     return value
 
 
+def _compute_default_max_backend_connections():
+    total_mem = psutil.virtual_memory().total
+    return int(total_mem / BYTES_OF_MEM_PER_CONN)
```

Comment:
```suggestion
    return max(int(total_mem / BYTES_OF_MEM_PER_CONN), 1)
```

or maybe the lower bound should be 2, idk.

Reply:
Good catch! As we do have a "pool", perhaps 2 makes more sense.